tests/integration_test/core_mqtt_system_test.c

```diff
@@ -883,11 +883,17 @@ void testSetUp()
 /* Called after each test method. */
 void testTearDown()
 {
+    MQTTStatus_t status;
+
     /* Terminate MQTT connection. */
-    TEST_ASSERT_EQUAL( MQTTSuccess, MQTT_Disconnect( &context ) );
+    status = MQTT_Disconnect( &context );
 
     /* Terminate TLS session and TCP connection. */
     ( void ) SecureSocketsTransport_Disconnect( &networkContext );
+
```

Comment:
Why are we throwing away the return value from `SecureSocketsTransport_Disconnect()` on line 892?